gen3utils/gitops/gitops_validator.py

```diff
@@ -391,3 +403,19 @@ def validate_against_dictionary(gitops, data_dictionary):
         )
 
     return ok
+
+
+def comment_gitops_errors_on_pr(repository, pull_request_number, gitops_errors):
+    token = os.environ["GITHUB_TOKEN"]
+    headers = {"Authorization": "token {}".format(token)}
+
+    repository = repository.strip("/")
+    base_url = "https://api.github.com/repos/{}".format(repository)
+    logger.info("Checking pull request: {} #{}".format(repository, pull_request_number))
+    pr_comments_url = "{}/issues/{}/comments".format(base_url, pull_request_number)
+    contents = ""
+    for error in gitops_errors:
+        contents += "## Gitops-etlMapping : {}".format(error)
+    full_comment = "# {}\n{}".format("gitops.json", contents)
+
+    submit_comment(full_comment, headers, pr_comments_url)
```

Comment:
this would deserve a nice little refactoring to put all the common code in an utils file 😄 can be done later though